blaze/utils.py

```diff
@@ -262,6 +278,27 @@ def register(typename, converter, converters=_converters):
 object_hook.register('datetime', pd.Timestamp)
 object_hook.register('frozenset', frozenset)
 object_hook.register('datashape', dshape)
+object_hook.register('builtin_function', lambda x: getattr(builtins, x))
+
+
+def numpy_pandas_function_from_str(f):
+    """
+    reconstruct function from string representation
+    """
+    if f.startswith("numpy"):
+        mod = np
+
+    elif f.startswith("pandas"):
+        mod = pd
+
+    else:
+        raise NotImplementedError("accepts numpy/pandas/builtin funcs only")
```

Comment:
Not sure if the server should raise an exception -- what is the best practice here? See resulting response object:

```
In [11]: expr = t.species.map(os.path.exists, 'str')
In [12]: response = test.post('/compute', data=serial.dumps(query), headers=mimetype(serial))
In [13]: response
Out[13]: <Response streamed [400 BAD REQUEST]>
```

@kwmsmith @llllllllll